.github/workflows/checks.yml

```diff
@@ -8,6 +8,8 @@ on:
 jobs:
   spotless:
     runs-on: ubuntu-latest
+    container: tmio/besu-build:1.0
```

Comment:
This should be a container in a hyperledger namespace instead of a user namespace.

This comment is for all references to containers in the `tmio` space, I don't want to litter this PR with comments.

Reply:
I certainly can't push to the hyperledger namespace - what's the procedure for that?

Reply:
I inquired about the process on discord. It is ongoing now.